docs/getStarted.rst

```diff
@@ -0,0 +1,177 @@
+Getting started
+================
+
+Even-though most of the routines in Avaframe are automatized, running them still
+requires a minimum amount of python scripting.
+Here we provide an example on how to perform simulations with Avaframe,
+but let's start with some information on the utilities to create logs and set the desired configurations.
+
+Utilities
+----------
+
```

Comment:
This is fine, but I suggest we have a 'in 5 minutes' (or similiarly named) section at the beginning, in which we have an enumerated list with the absolute bare minimum command and only the absolute necessary explanations to run a simulations. I.e.: 

- Clone the DFA repo by git clone...
- Clone repository by git clone...
- Change to directory i.e cd AvaFrame
- Local install by pip install ...
- Edit avaframe com2DFACfg.ini to 

and so on

Reply:
Included